pkg/eval/builtin_special.go

```diff
@@ -19,6 +19,7 @@ package eval
 import (
 	"os"
 	"path/filepath"
+	"plugin"
```

Comment:
This breaks the build on gccgo, which lacks the `plugin` package. It'd be nicer if gccgo has a `plugin` package with a stub implementation (I asked for that in https://github.com/golang/go/issues/36403#issuecomment-864474859), but for now, you'll need to manually stub out the plugin API like this:

plugin.go:

```go
//+build !gccgo

package eval

import "plugin"

var pluginOpen = plugin.Open
```

plugin_gccgo.go:

```go
//+build gccgo

package eval

import "errors"

var errPluginNotImplemented = errors.New("plugin not implemented")

type pluginStub struct{}

func pluginOpen(name string) (pluginStub, error) {
  return pluginStub{}, errPluginNotImplemented
}

func (pluginStub) Lookup(symName string) (interface{}, error) {
  return nil, errPluginNotImplemented
}
```